agent/stats/engine.go

```diff
@@ -441,6 +440,8 @@ func (engine *DockerStatsEngine) GetInstanceMetrics() (*ecstcs.MetricsMetadata,
 			continue
 		}
 
+		// TODO [SC]: Call GetServiceConnectStats() and add it to taskMetrics with corresponding taskArn
```

Comment:
Question; and potentially because I don't have context, is this todo supposed to be here?

Reply:
Thought GetInstanceMetrics() would be right place to gather service connect metrics since it initializes TaskMetric which contains the ServiceConnectMetricsWrapper.

```
type TaskMetric struct {
	_ struct{} `type:"structure"`

	ClusterArn *string `locationName:"clusterArn" type:"string"`

	ContainerMetrics []*ContainerMetric `locationName:"containerMetrics" type:"list"`

	ServiceConnectMetricsWrapper []*GeneralMetricsWrapper `locationName:"serviceConnectMetricsWrapper" type:"list"`

	...
}
```

Will implement this todo in next PR:)